### ISSUE TITLE
Accessibility issues in Grid

### ISSUE THREAD

**Issue by james-rae** [fgpv-vpgf/fgpv-vpgf#3805]
Two problems identified on the grid with respect to accessibility, provided by good pals at NrCan.

> There are no status messages to inform users that the data tables have been displayed on the page. Some users will not know that the page has been updated with the data table. This does not meet WCAG SC 4.1.3 Status Messages https://www.w3.org/WAI/WCAG21/Understanding/status-messages.html
Recommend using ARIA status role or live regions to indicate when the data table has finished loading

Also some of the grid columns are missing `role="columnheader"` attributes. They are needed to let a screenreader know what the columns are.
